tests/accuracy_utils.py

```diff
@@ -26,37 +26,83 @@
 sizes_pow_2 = [2**d for d in range(4, 11, 2)]
 sizes_noalign = [d + 17 for d in sizes_pow_2]
 sizes_1d = sizes_one + sizes_pow_2 + sizes_noalign
-sizes_2d_nc = [1, 16, 64, 1000]
-sizes_2d_nr = [1, 5, 1024]
+sizes_2d_nc = [1] if TO_CPU else [1, 16, 64, 1000]
+sizes_2d_nr = [1] if TO_CPU else [1, 5, 1024]
```

Comment:
`TO_CPU` indicates to compute the reference value on CPU, so use less and smaller sizes to reduce the runtime?

Reply:
> `TO_CPU` indicates to compute the reference value on CPU, so use less and smaller sizes to reduce the runtime?

Yes, we just pass the cpu tests, and will add cpu tests to ci tests soon. We will test all shapes when releasing a version.